src/tasks/minions/woodcuttingActivity.ts

```diff
@@ -15,10 +15,16 @@ export default class extends Task {
 		const { logID, quantity, userID, channelID, duration } = data;
 		const user = await this.client.fetchUser(userID);
 
-		const log = Woodcutting.Logs.find(Log => Log.id === logID)!;
+		const log = { ...deepClone(Woodcutting.Logs.find(Log => Log.id === logID))! };
```

Comment:
I'm really not a fan of this.  just do `let clueChance = log.clueChance;` and edit that variable instead.

Reply:
Resolved.